librepo/handle.c

```diff
@@ -812,6 +834,10 @@ lr_handle_setopt(LrHandle *handle,
         c_rc = curl_easy_setopt(c_h, CURLOPT_FILETIME, handle->preservetime);
         break;
 
+    case LRO_NETWORK_WAIT:
+        assert(lr_handle_network_wait(handle, err, va_arg(arg, long)));        
```

Comment:
Do we need this option at all anymore actually, if we're requiring the caller to invoke the API method directly?